### PR TITLE
Plugin E2E: Apply docs pr feedback

### PR DESCRIPTION
**Pull request by sunker**
**What this PR does / why we need it**:

[This](https://github.com/grafana/plugin-tools/pull/634) PR was merged before I had chance to apply @sympatheticmoose PR feedback. 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: packages/plugin-e2e/CONTRIBUTING.md [hunk 1]
```diff
@@ -137,6 +137,6 @@ yarn add @grafana/plugin-e2e@0.3.0-canary.623.aedff75.0
 yarn e2e:plugin
```
Comment (academo) [inline packages/plugin-e2e/CONTRIBUTING.md:137]: you also want to use `npm run` here.

Comment (sunker) [reply packages/plugin-e2e/CONTRIBUTING.md:137]: This (step 5) happend in grafana/grafana where we use yarn.

**Review by jackw — APPROVED** [review]
LGTM! 🚀

File: packages/plugin-e2e/CONTRIBUTING.md (hunk 1 above)
Comment (mckn) [reply packages/plugin-e2e/CONTRIBUTING.md:137]: I know that it is written in the text above but just to make it super clear we could add a comment above yarn commands. Something like:

```bash
# In your local grafana development folder.
yarn add @grafana/plugin-e2e@0.3.0-canary.623.aedff75.0
yarn e2e:plugin 
```

Comment (mckn) [reply packages/plugin-e2e/CONTRIBUTING.md:137]: I would also add something to make it super clear that the developer should discard the changes being made after running those commands until we have shipped a stable version of the plugin-e2e package.

**Review by mckn — COMMENTED** [review]
LGTM!

**Review by sympatheticmoose — APPROVED** [review]
Looks good - I also like the suggestions from Marcus 👍

**Pull request merged**